tools/clang/unittests/SPIRV/UtilsTest.cpp

```diff
@@ -0,0 +1,61 @@
+//===- unittests/SPIRV/UtilsTest.cpp ------ Utils tests -------------------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+#include "clang/SPIRV/Utils.h"
+#include "gtest/gtest.h"
+#include "gmock/gmock.h"
+
+namespace {
+
+using ::testing::ElementsAre;
+
```

Comment:
Need tests for empty string, string with one characters, two characters, three characters, and four characters. These are important corner cases.